nevergrad/optimization/differentialevolution.py

```diff
@@ -57,6 +59,17 @@ def twopoints(self, donor: np.ndarray, individual: np.ndarray) -> None:
             donor[: bounds[0]] = individual[: bounds[0]]
             donor[bounds[1] :] = individual[bounds[1] :]
 
+    def rotated_twopoints(self, donor: np.ndarray, individual: np.ndarray) -> None:
+        bounds = sorted(self.random_state.choice(donor.size + 1, size=2, replace=False).tolist())
+        if bounds[1] == donor.size and not bounds[0]:  # make sure there is at least one point crossover
+            bounds[self.random_state.randint(2)] = self.random_state.randint(1, donor.size)
+        bounds2 = sorted(
+            self.random_state.choice(donor.size + 1 - (bounds[1] - bounds[0]), size=1, replace=False).tolist()
+        )
```

Comment:
This is overly circumvoluted, do you mean:
```suggestion
        bounds2 = [self.random_state.choice(donor.size + 1 - (bounds[1] - bounds[0])]
```